### PR TITLE
Fix manifests and pylint

### PR DESCRIPTION
**Pull request by SilvioC2C**
Fix needed for #1403

### CODE REVIEW

File: partner_multi_relation/views/res_partner_relation_all.xml [hunk 1]
```diff
@@ -123,27 +123,4 @@
         </field>
     </record>
 
-    <record
```
Comment (NL66278) [inline partner_multi_relation/views/res_partner_relation_all.xml:126]: What is the reason for deleting this action?

Comment (SilvioC2C) [reply partner_multi_relation/views/res_partner_relation_all.xml:126]: It's a duplicate:
https://github.com/OCA/partner-contact/blob/12.0/partner_multi_relation/views/menu.xml#L6

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**